src/options/optionsMain.ts

```diff
@@ -130,6 +132,26 @@ export async function getOptions(rawOptions: any): Promise<AppOptions> {
     }
   }
 
+  if (rawOptions.wvvmp) {
+    process.env.ELECTRON_MIRROR =
+      'https://github.com/castlabs/electron-releases/releases/download/';
+    const electronVersion = options.packager.electronVersion;
```

Comment:
One last thing: did you verify your code works if the user doesn't pass a `--electron-version` ? Is `options.packager.electronVersion` already set at this point in this case?

Example: I'm sure your code works for `nativefier -e 11.0.2 --wvvmp`, but does it work for `nativefier --wvvmp` ?

Reply:
The tests I ran worked. If no `--electron-version` is passed, the try/catch block will abort the run on nonexisting versions.

Reply:
> The tests I ran worked. If no `--electron-version` is passed, the try/catch block will abort the run on nonexisting versions.

@sorhtyre hmmmm, I works, but I find it a bit :robot: . By not passing an `--electron-version`, a user expresses they don't care about the precise version, and they want whatever is the default for their version of Nativefier. So, in this case, I'd like your new Widevine code to use the Widevine build for this default version (currently, 10.1.6). So, rather than crashing and demanding an explicit version, I'd expect `nativefier --wvvmp https://...` to simply try to use `v10.1.6-wvvmp`. Does that make sense?

Reply:
Oops, maybe I didn't explain correctly... If no `--electron-version` is input by the user, the default version is passed. The real issue is that castLabs doesn't always produce a wvvmp version of Electron.

`nativefier --wvvmp https://...` does attempt `v10.1.6-wvvmp` but if it doesn't exist, nativefier will abort.

Do you want it to instead default to a hardcoded wvvmp version like it does with the normal version of Electron?

Reply:
Okay I misunderstood. No, we're good 👍, if the default with wvvmp doesn't exist, your error message will be clear to point users to go see what to pass.

Reply:
:+1: